### ISSUE TITLE
Project: Map-driven Street Sign Work Orders

### ISSUE THREAD

**Issue by johnclary** [cityofaustin/atd-data-tech#3163]
Enable transportation engineers to create street sign work orders from a mapping interface.

### Background and Context

This project encompasses our work to support ATD's citywide prima-facie speed limit conversion from 30 to 25 MPH. Our work includes IT support for the systematic collection of speed limit sign locations, roadway widths, and SMB Data Tracker enhancements to allow for the easy creation of work orders related to these sign locations.

### Resources
- [Link to folder in Drive](https://drive.google.com/drive/folders/1oCex8nnf9rgDg5IYeVegs4Rpg4rXMI4s)

---
This Github issue represents a project of Austin Transportation's [Data & Technology Services](https://data.mobility.austin.gov/about/) team. Project status is documented regularly in the comments below.

**Comment by johnclary** [comment]
My initial thoughts: I see three components to this work:

### Sign data capture and management

#### Needs
- Define a process to systematically survey all speed limit signs, and track progress as survey areas are complted.
- Ability to capture sign location and condition in the field

#### Solutions

- Thinking about the crosswalk data collection that @dianamartin did. Break the city into logical chunks (e.g. neighborhoods), assign them to collectors, mark them complete as the data is collected.
- Esri's new field collection tool. Can't remember what it's called, but Alan tested it out last year.
- Fulcrum
- Can we just use google street view?

## Street width data capture

#### Needs
- Ability to track which street widths have been collected
- Ability to store street widths somewhere (e.g., street segments dataset)

#### Solutions?
- Nearmap

### SMB Data Tracker / Work Orders 

#### Needs
-  ability to use existing points/have requesters drop points when submitting work order
- ability to automatically update asset status when work order is completed (e.g. from "Planned" to "Installed"

#### Solutions?
- New SMB Data Tracker UX for all of this

**Comment by amenity** [comment]
Status Update:
We made some minor modifications to the map layers last week, and engineers have started collecting points. One work order is ready to go, which John is using as his test case for finishing up the AGOL <> Data Tracker integration. Meeting schedule for 9/25 to review with Christina.
We will have to make a few adjustments based on discussion about DIGTESS tracking. Likely adding functionality to SMB Data Tracker to send DIGTESS notifications to TED staff.

**Comment by amenity** [comment]
@johnclary — 👆is from 9/22... anything since?

**Comment by johnclary** [comment]
We met again with TED to collection additional feedback. We're about 95% clear on what changes to be made to the integration. John needs to schedule a meeting with TED and SMB to confirm we're all in agreement, then John needs to update the integration. John is sorry this is taking a while.

**Comment by amenity** [comment]
@johnclary - do you think this will ultimately be covered by Moped?

**Comment by johnclary** [comment]
We have moved this project into the backlog after developing some basic arcgis online viewers for TED staff. We'll need to revisit this after #662

**Comment by johnclary** [comment]
This project never got out of the prototype phase and has been superseded by https://github.com/cityofaustin/atd-data-tech/issues/17301.
